lib/system.g

```diff
@@ -87,36 +97,34 @@ BIND_GLOBAL( "GAPInfo", rec(
       rec( short:= "A", default := false, help := ["disable/enable autoloading of suggested", "GAP packages"] ),
       rec( short:= "D", default := false, help := ["enable/disable debugging the loading of files"] ),
       rec( short:= "M", default := false, help := ["disable/enable loading of compiled modules"] ),
+      ,
+      rec( section:= ["TODO FIXME"] ),
       rec( short:= "N", default := false, help := ["do not use hidden implications"] ),
       rec( short:= "O", default := false, help := ["disable/enable loading of obsolete files"] ),
       rec( short:= "T", long := "nobreakloop", default := false, help := ["disable/enable break loop and error traceback"] ),
       rec( long := "alwaystrace", default := false, help := ["always print error traceback", "(overrides behaviour of -T)"] ),
       rec( long := "quitonbreak", default := false, help := ["quit GAP with non-zero return value instead", "of entering break loop"]),
       ,
-      rec( short:= "L", default := "", arg := "<file>", help := [ "restore a saved workspace", "(available only if GAP uses GASMAN)"] ),
+      rec( section:= ["Workspaces:", "  (only available if GAP uses GASMAN)"] ),
+      rec( short:= "L", default := "", arg := "<file>", help := [ "restore a saved workspace"] ),
       rec( short:= "R", default := false, help := ["prevent restoring of workspace (ignoring -L)"] ),
       ,
-      rec( short:= "p", default := false, help := ["enable/disable package output mode"] ),
-      rec( short := "E", default :=false ),
-      rec( short := "s", default := "4g" ),
```

Comment:
-E and -s are already mentioned elsewhere, and with a default value, so these two lines didn't do anything... I think... ???